### DISCUSSION TITLE
Nomenclature for inferred junctions

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#15] [HGVS nomenclature proposals]
The VICC gene fusion community wants to explicitly annotate when a fusion is inferred. They have [provided a nomenclature for this](https://fusions.cancervariants.org/en/latest/nomenclature.html#inferred-fusions), adding parentheses around double-colons ( `(::)` ) to indicate an inferred break/join.

We would like to align this with the recommendations of the HVNC and incorporate the committee recommendations into the HGVS specification.

**Comment by jtdendunnen** [comment]
I wonder whether the format would be used in HGVS nomenclature. Please provide an example HGVS variant description including the proposed '(::)".

**Reply by ahwagner** [reply]
When directly detected, the following BCR::ABL1 fusion transcript ([COSMIC fusion](https://cancer.sanger.ac.uk/cosmic/fusion/summary?id=1755)) is often seen in chronic myelogenous leukemia (CML):

ENST00000305877.12:r.1_3533::ENST00000318560.5:r.461_5766

But let's say that there was a WGS assay performed and a reciprocal translocation between chr22 and chr9 was detected, defined by breakpoints chr22:23291000 and chr9:130853500. This assay was performed in a diagnostic setting for a tumor suspected to be CML, and the presence of this rearrangement is compatible with the often-observed fusion above. Reporting the inferred presence of this fusion would look like:

ENST00000305877.12:r.?\_3533(::)ENST00000318560.5:r.461\_? (junction only)
ENST00000305877.12:r.1_3533(::)ENST00000318560.5:r.461_5766 (full transcript segments)

using the current HGVS fusion transcript syntax.

**Reply by marinadistefano** [reply]
I am in favor of adopting compatible recommendations as long as the cytogenetics community is on board.

**Comment by jtdendunnen** [comment]
Would following current recommendations these descriptions not be:

`(ENST00000305877.12:r.?_3533::ENST00000318560.5:r.461_?)`
`(ENST00000305877.12:r.1_3533::ENST00000318560.5:r.461_5766)`

**Reply by ahwagner** [reply]
@RosHastir mentioned the following challenges:

1. Uncertainty about breakpoints
2. Uncertainty about existence of fusion
3. Both of the above

@ahwagner mentioned that there are conflicts with the above and the use of the english language.

@jtdendunnen thinks the proposed syntax `(::)` for describing whether or not the fusion exists is probably good.

@ahwagner to come up with some examples resolving the three above situations.
